services/api/db/urls.py

```diff
@@ -0,0 +1,20 @@
+from django.urls import path
+from db.views import ProfileViewSet, OrganizationSettingsViewSet
+from rest_framework.urlpatterns import format_suffix_patterns
+
+
+urlpatterns = [
+    path('bio/', ProfileViewSet.as_view({'get': 'bio', 'post': 'bio', 'put': 'bio', 'delete': 'bio'})),
+    path('date_of_birth/', ProfileViewSet.as_view({'get': 'date_of_birth', 'post': 'date_of_birth', 'put': 'date_of_birth', 'delete': 'date_of_birth'})),
+    path('profile_image/', ProfileViewSet.as_view({'get': 'profile_image', 'post': 'profile_image', 'put': 'profile_image', 'delete': 'profile_image'})),
```

Comment:
Why do we need separare endpoints for each urls.

Reply:
i could provide single url for updating these fields but i thought since its profile. Its better to have separate url for better readability by front end.....should i change it?

Reply:
also it is much easy to retrieve each user profile field

Reply:
We will go with the simplest approach, ie., one API for the whole entity. If we need this definitive requirement, we can split.

Reply:
ohkiee i will update it

Reply:
you could look at new urls endpoint , i guess it will solve the doubts